pkg/infra/process.go

```diff
@@ -3,15 +3,23 @@ package infra
 import (
 	"fmt"
 	"os"
-	"strconv"
 	"time"
 
 	"github.com/pkg/errors"
 	log "github.com/sirupsen/logrus"
+	"gopkg.in/alecthomas/kingpin.v2"
 )
 
 const loglevel = "TAPE_LOGLEVEL"
 
+var (
+	app = kingpin.New("tape", "A tiny, handy tool to test performance of a deployed Fabric network")
+
+	run = app.Command("run", "Start the tape program").Default()
+	con = run.Flag("config", "Set the config file's path").Default("./config.yaml").Short('c').String()
+	num = run.Flag("number", "Set the number of sending tx").Default("1000").Short('n').Int()
```

Comment:
number of sending tx.../ number of tx for shot?

Reply:
Yeah, I will use this description statement.